README.md

```diff
@@ -55,8 +55,8 @@ Getting started
 
     ```ruby
     require 'simplecov'
-    SimpleCov.start
-
+    SimpleCov.start # Or SimpleCov.start 'rails'
+    
```

Comment:
The start of simplecov with rails built-in configuration is a little hard to find in the Readme. We should disclose it in a different fashion.